src/intersections/lines.jl

```diff
@@ -63,7 +63,7 @@ function intersectparameters(a::Point{Dim,T}, b::Point{Dim,T},
 
   # calculate parameters of intersection or closest point
   if r ≥ 2
-    λ = A \ y
+    λ = A \ SVector(y.coords)
```

Comment:
This one we thought it would be a good idea to convert both A and y in the beginning of the function right?

Reply:
`A` is already a `SMatrix`, and I am now tranforming `y` from the beginning. Check the updated version please.